data/campaigns/Dead_Water/scenarios/05_Tirigaz.cfg

```diff
@@ -748,6 +761,21 @@
                     {NEW_GOLD_CARRYOVER 40}
                 [/endlevel]
             [/then]
+        
+            [else]
+                [objectives]
+                    side=1
+                    {HOW_TO_LOSE}
+                    [objective]
+                        description= _ "Destroy all the undead"+{EARLY_FINISH_BONUS_FOOTNOTE}
+                        condition=win
+                    [/objective]
+
+                    [gold_carryover]
+                        carryover_percentage=40
+                    [/gold_carryover]
+                [/objectives]
```

Comment:
I would prefer if you could instead of a new [objectives] block edit the first one like it is done here:
https://github.com/wesnoth/wesnoth/blob/1.14/data/campaigns/The_South_Guard/scenarios/07a_Into_the_Depths.cfg#L299-L309
As we have not a variable here, the condition could be
```
[show_if]
    [have_unit]
        side=number of the side
        canrecruit=yes
    [/have_unit]
[/show_if]
```

This second [objectives] tag could be replaced by
`[show_objectives][/show_objectives]` then